### PR TITLE
Update to JSON API version 1.3 

### PR DESCRIPTION
**Pull request by yoink00**
Changes:
 * Updates "no change" endpoints to 1.3
 * Updates existing but changed endpoints to 1.3:
    * Tags
    * Account
    * Server
    * IPAddress

There will be some tweaks once the "added" resources are complete.

### CODE REVIEW

File: upcloud/service/service.go [hunk 1]
```diff
@@ -707,7 +711,7 @@ func (s *Service) GetTags() (*upcloud.Tags, error) {
 }
 
 // Wrapper that performs a GET request to the specified location and returns the response or a service error
-func (s *Service) basicJSONGetRequest(location string) ([]byte, error) {
+func (s *Service) basicFutureGetRequest(location string) ([]byte, error) {
```
Comment (PopoSensei) [inline upcloud/service/service.go:714]: Why future and not just basicGetRequest?

Comment (PopoSensei) [reply upcloud/service/service.go:714]: And why change this at all?

File: examples/upcloud-cli/main.go [hunk 2]
```diff
@@ -87,10 +87,16 @@ func createServer(s *service.Service) error {
...
-				Family: upcloud.IPAddressFamilyIPv4,
+		Networking: &request.CreateServerNetworking{
+			Interfaces: []request.CreateServerInterface{
+				{
+					IPAddresses: []request.CreateServerIPAddress{
+						{
+							Family: upcloud.IPAddressFamilyIPv4,
+						},
+					},
+					Type: upcloud.IPAddressAccessUtility,
```
Comment (aakso) [inline examples/upcloud-cli/main.go:98]: I'd suggest not to use constants from 1.2 here as the interface types in 1.3 are different thing.

File: upcloud/service/service.go (hunk 1 above)
Comment (yoink00) [reply upcloud/service/service.go:714]: Hi,

This is a bit of a hangover from the way I did the port. `basicFutureGetRequest` was for the `1.3` API and `basicJSONGetRequest` was for the `1.2` API.

I removed all references of `future` in the code once I finished but I missed this one. I spotted it yesterday and it's already fixed in the next batch of changes.

Would you like me to fix it now or are you happy to wait for the next PR?

Thanks,

Stuart

File: examples/upcloud-cli/main.go (hunk 2 above)
Comment (yoink00) [reply examples/upcloud-cli/main.go:98]: Hi,

Yes, I've spotted that too. I've added appropriate networking constants in the next batch of changes and switched everything to those.

Would you like me to fix it now or are you happy to wait for the next PR?

Thanks,

Stuart

Comment (aakso) [reply examples/upcloud-cli/main.go:98]: Ok just wanting to check :) Let's get back to this in next PR then.

File: upcloud/service/service.go (hunk 1 above)
Comment (PopoSensei) [reply upcloud/service/service.go:714]: It's ok, for the next PR. 👍

**Pull request merged**